packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -296,6 +300,16 @@ class InteractiveViewer extends StatefulWidget {
   /// than maxScale.
   final double minScale;
 
+  /// Changes the scrolling behavior after end of a gesture.
```

Comment:
Nit: 

```suggestion
  /// Changes the deceleration behavior after a gesture.
```

Just avoiding future confusion about the interactive viewer not actually being a scrolling widget :)